File: src/problems/generate-parentheses/boilerplate-full/function.java
//USER_CODE_HERE
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        Solution sol = new Solution();
        List<String> result = sol.generateParenthesis(n);
        System.out.println(result);
    }
}

File: src/problems/generate-parentheses/boilerplate-full/function.py
#USER_CODE_HERE
import sys
from typing import List

# Set recursion limit higher for recursive problems
sys.setrecursionlimit(2000)

if __name__ == "__main__":
    # Input reading for the specific problem
    n = int(input())

    sol = Solution()
    
    # Call the solution function
    result = sol.generateParenthesis(n)
    
    # Print the result
    print(result)